components/bpp-recv/powerdown.h

```diff
@@ -3,10 +3,15 @@
 
 #define POWERDOWN_DBG 1
 
+typedef enum {
+	POWER_MODE_BPP=0,
+	POWER_MODE_UPY,
+	POWER_MODE_MAX
```

Comment:
I think _MAX should be maximum value (here POWER_MODE_UPY).
Something like POWER_MODE_ELTS, POWER_MODE_COUNT or POWER_MODE_NUM sounds better.

Or set POWER_MODE_MAX to POWER_MODE_UPY:

    typedef enum {
      POWER_MODE_BPP=0,
      POWER_MODE_UPY,
      POWER_MODE_MAX=POWER_MODE_UPY
    } PowerMode;